e2e/nomostest/gitproviders/bitbucket.go

```diff
@@ -229,6 +229,7 @@ func filterObsoleteRepos(bytes []byte) ([]string, int, error) {
 	nextPage := -1
 	var response interface{}
 	if err := json.Unmarshal(bytes, &response); err != nil {
+		fmt.Printf("error unmarshalling json in filterObsoleteRepos:\n%v\n", bytes)
```

Comment:
Printf?
Can we inject the Logger?

Reply:
I went this route since the file is already using Printf, but we could inject the logger

Reply:
refactored to inject the logger